Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,20 @@
 import React from 'react';
 import './App.css';
+import ButtonContainer  from './components/ButtonComponents/ButtonContainer';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
 
 const App = () => {
+
+  const display = {
+    text: 0,
+    style: "output-display"
+  };
+
+
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="container">
+      <CalculatorDisplay number={display} class={display}/>
```

Comment:
This should be className instead of class. Also why are you giving the display a className of whatever number it is displaying?